tests/dragonfly/replication_test.py

```diff
@@ -1344,6 +1344,9 @@ async def test_tls_replication(
 
     # 4. Kill master, spin it up and see if replica reconnects
     master.stop(kill=True)
+    # We need to await, because kill sends a SIGTERM which can then be ignored or blocked
+    # By waiting 1 sec it gives enough time for the signal to be handled by the process
+    await asyncio.sleep(10)
```

Comment:
calling await proc.wait() after proc.kill() waits for the process to exit
do you think we can use it here instead of sleep